freeipa/src/main/resources/freeipa-salt/pillar/freeipa/init.sls

```diff
@@ -4,3 +4,11 @@ freeipa:
   realm: testrealm
   admin_user: admin
   freeipa_to_replicate:
+  backup:
+    enabled: true
```

Comment:
I would set it here to false, and override it from java if enabled

Reply:
The desire from PM and exec is to have the default be enabled.  We don't have mechanisms in the UI to enable this so enabling for customers will be difficult.

Reply:
enable by default from java. usually we place empty or test values into pillars so it won't contain anything that could cause wrong behavior. I just want to be sure that when this feature is enabled, it's intentional. Like if somebody touch the code and doesn't push down the pillar related to backup when it is disabled, it would cause backup being enabled, because the value here says it.

Reply:
changed.